docs/api/api.md

```diff
@@ -58,10 +58,19 @@ import "github.com/cert-manager/trust-manager/pkg/apis/trust/v1alpha1"
 
 ## Constants
 
-<a name="BundleConditionSynced"></a>
```

Comment:
I guess this anchor link is just wrong, right? Seems to be a gomarkdoc bug - this should surely be "constants" or something similar, not the name of one of the constants.

Not a blocker obviously, just interesting!